### ISSUE TITLE
multi-row_tabs: add a shortcut to collapse the rows

### ISSUE THREAD

**Issue by j1warren** [MrOtherGuy/firefox-csshacks#73]
Is that technically possible? Fully collapse or change the row count to 1 or 2 (preferably).

Situation:
multi-row tabs take some vertical space (e.g. I have 10) and it's ok.
But sometimes I wanna free the vertical space to read an article, but I don't want to F11 the FF window to the whole screen.

**Comment by MrOtherGuy** [comment]
10 rows!? What the... Reminds me of IE and toolbars in the early 2000's - not a happy memory ;)

But no, unfortunately such a thing (essentially creating a checkbox) is not possible with CSS alone.

The behavior is possible but it would require an extension to help. Any extension that adds/removes some known prefix to/from the window title would do the trick. Like, you click its button and it adds a prefix, click again and it removes it. Then you can use the prefix information to set max-rows in CSS. It's a bit convoluted but works fine.

**Comment by j1warren** [comment]
Sorry for bump. Just a chrome question from a newbie, is it possible from an extension to edit variable `--multirow-n-rows` of `multi-row_tabs.css`?

**Comment by MrOtherGuy** [comment]
No, it's not possible - but the method I described in the previous comment should work fine.
